test.sh

```diff
@@ -214,6 +214,7 @@ print_heading "Settings:"
 trap "print_outcome" EXIT
 
 settings="$(cat -- <<-EOM
+    TRAVIS_GO_VERSION:  ${TRAVIS_GO_VERSION-not specified}
```

Comment:
```suggestion
    TRAVIS_GO_VERSION:  ${TRAVIS_GO_VERSION:-not specified}
```

Reply:
By the way, one thing I noticed in here: You assign the here doc to `$settings`, and then immediately print it. You could simplify slightly:

```
print_heading "Settings:"
cat <<-EOM
  TRAVIS_GO_VERSION: ...
EOM
print_heading "Starting..."
```

Reply:
That's funny! I didn't notice that. I've since pulled the [experimental] change on this line, so I'm going to pass on this for now.